src/tools/ips.rs

```diff
@@ -0,0 +1,8 @@
+// If the IPS array is empty then it means that source IP addresses have not been
+// generated and assigned to dummy devices. In such case tests should not bound to any
+// specific address and use local pool instead.
+
+/// Reference to a static array of IP addresses (represented as str).
+/// If array is empty generate new addresses using eg.:
+/// python3 tools/ips.py --subnet 1.1.1.0/28 --file src/tools/ips.rs --dev_prefix test_dev
+pub const IPS: &[&str] = &[];
```

Comment:
I can't do this on Mac, what's the size of this file after we do this?

Reply:
Would it be possible to use `ifconfig` instead here so it also works on Mac?

Reply:
Yes, it'd be possible to use ifconfig on Mac (although you can use brew to install ip -> iproute2mac), but either one does not allow to create dummy devices.
Probably for Macs there will be a need to either create address aliases for single interface or add other type of interfaces (maybe tap and bridge it).